### PR TITLE
prokka 1.12_1 (remove bundled binaries + rpath-ing)

### PR DESCRIPTION
**Pull request by tseemann**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by tseemann** [comment]
```
homebrew/science/prokka:
  * perl modules should be vendored rather than use deprecated `depends_on "Bio::Perl" => :perl`
  * perl modules should be vendored rather than use deprecated `depends_on "XML::Simple" => :perl`
  * perl modules should be vendored rather than use deprecated `depends_on "Time::Piece" => :perl`
```

File: prokka.rb [hunk 1]
```diff
@@ -1,11 +1,14 @@
 class Prokka < Formula
   desc "Rapid annotation of prokaryotic genomes"
   homepage "http://www.vicbioinformatics.com/software.prokka.shtml"
+  # doi "10.1093/bioinformatics/btu153"
+  # tag "bioinformatics"
+
```
Comment (sjackman) [inline prokka.rb]: Can you revert this change please? The comments go after `head`. (It was the other way around before.)

File: prokka.rb [hunk 2]
```diff
@@ -19,37 +22,14 @@ class Prokka < Formula
   depends_on "tbl2asn"
   depends_on "parallel"
   depends_on "minced"
+  depends_on "barrnap"
```
Comment (sjackman) [inline prokka.rb]: Can you please put these dependencies in alphabetical order?

**Comment by sjackman** [comment]
Thanks, Torsten. Looks good to me (save my requested aesthetic changes).
I'm okay ignoring the vendored audit warnings for now. I'd like to create a formula for bioperl. Five formula depend on `Bio::Perl`.

**Comment by sjackman** [comment]
@tmozgach Can you please test this PR in ORCA?
```sh
brew upgrade https://raw.githubusercontent.com/tseemann/homebrew-science/4201dd5bbd8c5c33dc03edef30510acf58205a2c/prokka.rb
brew test prokka
```

**Comment by tmozgach** [comment]
@sjackman works

**Comment by tseemann** [comment]
@sjackman @tmozgach ok i've made the cosmetic changes.

I will open an issue for `Bio::Perl` to discuss.

**Comment by tseemann** [comment]
```
brew pull --bottle https://github.com/Homebrew/homebrew-science/pull/621
error: could not lock config file /home/linuxbrew/.linuxbrew/.git/config: Permission denied
==> Fetching patch
Patch: https://github.com/Homebrew/homebrew-science/pull/621.patch
error: cannot open .git/FETCH_HEAD: Permission denied
Error: Failure while executing: git fetch --quiet origin pull/621/head
```
@sjackman i was afraid of this... any ideas?
API or BINTRAY tokens?  
shallow github or something?

**Comment by tseemann** [comment]
Why does this always fail for me?

```
brew pull --bottle https://github.com/Homebrew/homebrew-science/pull/621
==> Fetching patch
Patch: https://github.com/Homebrew/homebrew-science/pull/621.patch
######################################################################## 100.0%
==> Applying patch
Applying: Bump igv version to 2.3.25
Using index info to reconstruct a base tree...
M	igv.rb
Falling back to patching base and 3-way merge...
Auto-merging igv.rb
CONFLICT (content): Merge conflict in igv.rb
error: Failed to merge in the changes.
Patch failed at 0001 Bump igv version to 2.3.25
The copy of the patch that failed is found in: .git/rebase-apply/patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
Error: Patch failed to apply: aborted.
```
